srml/metadata/src/lib.rs

```diff
@@ -261,7 +261,13 @@ pub enum StorageFunctionType {
 	Map {
 		key: DecodeDifferentStr,
 		value: DecodeDifferentStr,
-	}
+	},
+	DoubleMap {
```

Comment:
hmm this requires to bump metadata version like https://github.com/paritytech/substrate/pull/2037 no ?

Reply:
:+1: I think so.